cpp/marc_tools/bash_script_convert_core_json_data.sh

```diff
@@ -1,14 +1,14 @@
-#! /bin/bash
+#!/bin/bash
```

Comment:
Please rename this file and remove the bash_script_ prefix!  All our files ending in ".sh" are Bash scripts.